src/ADAL.PCL/Internal/Telemetry/EventConstants.cs

```diff
@@ -0,0 +1,114 @@
+        public const string CorrelationId = "correlation_id";
+        public const string GivenName = "given_name";
+        public const string DisplayableId = "displayable_id";
+        public const string UniqueId = "unique_id";
+        public const string UserAgent = "user_agent";
+        public const string RequestApiVersion = "request_api_version";
+        public const string HttpBodyParameters = "http_body_parameters";
+        public const string HttpResponseMethod = "http_response_method";
+        public const string IsMRRT = "is_MRRT";
+        public const string ExpiredAt = "expired_at";
+        public const string TokenFound = "token_found";
+        public const string CacheLookUp = "cache_lookup";
+        public const string LoginHint = "login_hint";
+        public const string HttpQueryParameters = "query_parameters";
+        public const string HttpStatusCode = "response_code";
+        public const string IsDeprecated = "is_deprecated";
+        public const string ExtendedExpires = "extended_expires_on_setting";
+        public const string UserCancel = "user_cancel";
+        public const string HttpEventCount = "http_event_count";
+
+        public const string AcquireTokenSilentAsync1 = "3";
+        public const string AcquireTokenSilentAsync2 = "9";
+        public const string AcquireTokenSilentAsyncClientCredential = "10";
+        public const string AcquireTokenSilentAsyncClientCertificate = "11";
+        public const string AcquireTokenSilentAsyncClientAssertion = "12";
+
+        public const string AcquireTokenAsyncInteractive1 = "140";
+        public const string AcquireTokenAsyncInteractive2 = "146";
+        public const string AcquireTokenAsyncInteractive3 = "151";
+
+        public const string AcquireTokenOnBehalfOf = "500";
+        public const string AcquireTokenOnBehalfOfClientCertificate = "506";
+        public const string AcquireTokenOnBehalfOfClientAssertion = "511";
+
+        public const string AcquireDeviceCodeAsync = "606";
+        public const string AcquireTokenByDeviceCodeAsync = "611";
+
+        public const string AcquireTokenAsyncUserAssertion = "700";
+        public const string AcquireTokenAsyncClientCredential = "706";
+        public const string AcquireTokenAsyncClientCertificate = "711";
+        public const string AcquireTokenAsyncClientAssertion = "716";
+
+        public const string AcquireTokenByAuthorizationCodeAsyncClientCredential1 = "800";
+        public const string AcquireTokenByAuthorizationCodeAsyncClientCredential2 = "806";
+        public const string AcquireTokenByAuthorizationCodeAsyncClientAssertion1 = "811";
+        public const string AcquireTokenByAuthorizationCodeAsyncClientAssertion2 = "816";
+        public const string AcquireTokenByAuthorizationCodeAsyncClientCertificate1 = "821";
+        public const string AcquireTokenByAuthorizationCodeAsyncClientCertificate2 = "826";
+
+        public const string AcquireTokenCommonAsync = "722";
```

Comment:
>AcquireTokenCommonAsync [](start = 28, length = 23)

AcquireTokenCommon is a internal API and not a part of the public surface. Please rename this to correct API name.